src/phantom_token.c

```diff
@@ -512,11 +577,15 @@ static ngx_int_t introspection_response_handler(ngx_http_request_t *request, voi
 
     module_context->status = request->headers_out.status;
 
-    // fail early for not 200 response
+    // Fail early for not 200 response
     if (request->headers_out.status != NGX_HTTP_OK)
     {
+        // Log any error except 204 expiry responses, which are expected
+        if (request->headers_out.status != NGX_HTTP_NO_CONTENT) {
+            ngx_log_error(NGX_LOG_ERR, request->connection->log, 0, "Introspection subrequest returned response code: %d", request->headers_out.status);
```

Comment:
This would mean, for example, that the configured credentials for introspection are invalid, right? Shouldn't happen in regular usage, including expired tokens.

Reply:
We had some issues where at times the caller received a 502 with nothing in logs. So this will ensure that invalid configurations are highlighted and that is my intent.

Reply:
Right, just wanted to be sure we don't log ERR in regular usages. Seems OK.